Blog/the power of integrity in both work and life (#275)
iodigital-com/io-technology: data/blog/the-power-of-integrity-in-both-work-and-life.md

---
title: 'The power of integrity in both work and life'
date: '2025-01-06'
tags: ['integrity', 'leadership', 'personal-development']

> images: ['/articles/the-power-of-integrity-in-both-work-and-life/banner.png']

Reviewer: For this image, can you remove the iO logo and the link on the picture? We have automatic OG images generated in iO style on prod :)

summary: 'In this article, I explain how being a person of integrity has helped me. I share my experiences and tips to help you on your journey towards integrity in both personal and professional life.'
authors: ['komeil-mehranfar']
theme: 'blue'
---

## Introduction

> Eight months ago, I took a leadership course inspired by EST (Erhard Seminar Training), and it changed my life. It helped me improve my relationship with my wife, my work at "iO", my project "Lyreco" (an iO client), staying fit, publishing my [Sudoku app](https://sudoku.best/), and starting a basketball community for iO employees and clients. It was one of the best investments I've made, and it reinforced my goal of becoming a great leader. To achieve this, I need to master self-leadership by staying organised and keeping my promises.

Reviewer: ```suggestion
Eight months ago, I took a leadership course inspired by EST (Erhard Seminar Training), and it changed my life. It helped me improve my relationship with my wife, my work at iO, my project Lyreco (an iO client), staying fit, publishing my [Sudoku app](https://sudoku.best/), and starting a basketball community for iO employees and clients. It was one of the best investments I've made, and it reinforced my goal of becoming a great leader. To achieve this, I need to master self-leadership by staying organised and keeping my promises.
```

The problem is, just hearing or reading these messages doesn't help me become a person with integrity. It's not actionable advice.

So, how do I create a clear path to integrity? In this article, I'll introduce a conversational framework that provides direct access to integrity. I recommend reading the primary article I've linked above as well.

Let's start with how integrity is defined in that document.

## Definition of integrity

"Integrity is defined as _a state or condition of being whole, complete, unbroken, unimpaired, sound, and in perfect condition._"

But what does _integrity_ mean for individuals?

According to this model:

> "In this new model, integrity for a person is a matter of that person's word, nothing more and nothing less. Be it my word to myself (e.g., making a promise to myself, or a comment to myself about myself), or my word to others, in fact, it is my word through which I define and express myself, both for myself and for others."

So, in simple terms, personal integrity means **honouring your word.**

But what does it actually mean to "honour one's word"? Let's explore that.

## The two wings of honouring one's word

> "In this new model of integrity we define honouring your word as:

Reviewer: ```suggestion
In this new model of integrity we define honouring your word as:
```

1. Keeping your word (and on time).

And, whenever you will not be keeping your word:

> 2. Just as soon as you become aware that you will not be keeping your word (including not keeping your word on time) saying to everyone impacted

Reviewer: line 48-52 don't flow so well. It's difficult to read. Perhaps don't make it a numbered list and just write the copy out

   1. that you will not be keeping your word, and
   2. that you will keep that word in the future, and by when, or that you won't be keeping that word at all, and

>    3. what you will do to deal with the impact on others of the failure to keep your word (or to keep it on time)."

Reviewer: ```suggestion
   3. what you will do to deal with the impact on others of the failure to keep your word (or to keep it on time).
```

Another common scenario is when people ask about your task delivery timeline, and your responses are filled with conditions and vagueness like, "I can't say for sure. Maybe today or tomorrow. If there's no issue with the API or design, I'll deliver."

### **1. Keeping Your Word (and on time):**

**Example:**

You are given a task to add a new login feature to an app and promise to finish it by the end of the week. You plan your time well, focus on this task all week, and complete it by Friday as you said you would. This shows you kept your promise and met the deadline.

### **2. Communicating When You Will Not Be Keeping Your Word:**

**Example:**

You're working on a feature update for a client, with a deadline set for Wednesday. Halfway through the project, you find a major bug that causes a big delay. As soon as you know you can't meet the deadline, you quickly tell your project manager and the client. You explain:

1. You won't be able to deliver the update by Wednesday.
2. You give a new completion date, explaining how long it will take to fix the issue and finish the project.

> 3. To lessen the impact, you offer a temporary solution or focus on fixing the critical bug to reduce disruptions for the client until the full update is ready.

Reviewer: These two headings with copy seem a bit out of place. Can you look at it again and see how you can rewrite it to flow better in the article?

> ## One's word definition

Reviewer: ```suggestion
## The definition of one's word
```

> "In this new model of integrity, we define a person's word as consisting of each of the following:

Reviewer: ```suggestion
In this new model of integrity, we define a person's word as consisting of each of the following:
```

> 1. What You Said: Whatever you have said you will do or will not do, and in the case of do, by when you said you would do it.

Reviewer: This sentence is very difficult to read. Can you split it up into multiple sentences so it's easier?

> 2. What You Know: Whatever you know to do or know not to do, and in the case of do, doing it as you know it is meant to be done and doing it on time, unless you have explicitly said to the contrary.

Reviewer: This sentence is very difficult to read. Can you split it up into multiple sentences so it's easier?

3. What Is Expected of you by those with whom you wish to have a workable relationship (unless you have explicitly declined those unexpressed requests).
4. What You Say Is So: Whenever you have given your word to others as to the existence of some thing or some state of the world, your word includes being willing to be held accountable that the others would find your evidence for what you have asserted also makes what you have asserted valid for themselves.

> 5. What You Stand For: What you stand for is fundamental to who you are for yourself and who you are for others. What you stand for is a declaration constituted by 1) who you hold yourself to be for yourself as that for which you can be counted on from yourself (whether specifically articulated by you or not), and 2) who you hold yourself out to be for others as that for which you can be counted on by others (or have allowed others to believe as that for which you can be counted on).

Reviewer: This sentence is very difficult to read. Can you split it up into multiple sentences so it's easier?

> 6. Moral, Ethical And Legal Standards: The social moral standards, the group ethical standards and the governmental legal standards of right and wrong, good and bad behaviour, in the society, groups and state in which one enjoys the benefits of membership are also part of one's word (what one is expected to do) unless a) one has explicitly and publicly expressed an intention to not keep one or more of these standards, and b) one is willing to bear the costs of refusing to conform to these standards (the rules of the game one is in)."

Reviewer: This sentence is very difficult to read. Can you split it up into multiple sentences so it's easier?

2. **What You Know:**
   - You know you need to write tests for every new piece of code to make sure it works well.
   - You know not to merge your code into the main branch until all issues from the review are fixed.
3. **What Is Expected of You:**
   - Your team expects you to join daily meetings to share your progress.
   - Your clients expect you to keep the app running during business hours as agreed.
4. **What You Say Is So:**
   - You told your team the new module works well with the current system.
   - You said the recent software bug is fixed and showed test results to prove it.
5. **What You Stand For:**
   - You believe in delivering high-quality code, making sure your projects are free from avoidable mistakes.
   - You see yourself as someone who values teamwork, always ready to help colleagues.
6. **Moral, Ethical, and Legal Standards:**
   - You follow the company's coding rules, making sure not to copy open-source code without giving credit.
   - You follow data protection laws, ensuring user data is handled according to GDPR when working on apps for EU users.

## **Eleven hidden factors hiding out-of-integrity behaviour**

> 

Reviewer: Might be nice to add a bit of copy here. For instance, something like "Let's have a look at hidden factors that...." It helps the flow of the story

1. **Not seeing that who you are as a person is your word**

>    "That is, thinking that who you are as a person is anything other than your word. For

Reviewer: ```suggestion
   That is, thinking that who you are as a person is anything other than your word. For
```

   example, thinking that who you are is your body, or what is going on with you
   internally (your mental/emotional state, your thoughts/thought processes and your
   bodily sensations), or anything else you identify with such as your title or position in
   life, or your possessions, etc… leaves you unable to see that when your word is less
   than whole and complete you are diminished as a person.
   A person is constituted in language. As such, when a person's word is less than whole

>    and complete they are diminished as a person."

Reviewer: ```suggestion
   and complete they are diminished as a person.
```

   **Example:**

   Alex is a software engineer who is very good at coding. He believes being a tech expert defines him.

   Alex promises to finish a new software feature by the end of the month. But he gets too focused on perfecting the code and misses the deadline. He thinks it's okay because he's the "expert."

   Alex forgets that keeping promises is important. By missing the deadline, people start to trust him less, no matter how good he is at coding. Not keeping his word makes him seem unreliable to others.

> 2. **Living as if my Word is only What I Said (Word 1) and What I Assert Is True (Word 4)**

Reviewer: What are these "word 1" and "word 4"? They seem to come a bit out of the blue. Also, the casing of this title is inconsistent and a bit difficult to understand

Reviewer: Perhaps just leave the "(word 1)" and "(word 4)" out of the title here and add them in the list below

>    "Even if we are clear that in the matter of integrity our word exists in six distinct ways, most of us actually function as if our word consists only of _what I said_ or _what I assert_ _is true_. This guarantees that we cannot be men or women of integrity. For us, Words 2, 3, 5, and 6 are invisible as our word:

Reviewer: ```suggestion
   Even if we are clear that in the matter of integrity our word exists in six distinct ways, most of us actually function as if our word consists only of _what I said_ or _what I assert_ _is true_. This guarantees that we cannot be men or women of integrity. For us, Words 2, 3, 5, and 6 are invisible as our word:
```

>    "Even if we are clear that in the matter of integrity our word exists in six distinct ways, most of us actually function as if our word consists only of _what I said_ or _what I assert_ _is true_. This guarantees that we cannot be men or women of integrity. For us, Words 2, 3, 5, and 6 are invisible as our word:

Reviewer: ```suggestion
   "Even if we are clear that in the matter of integrity our word exists in six distinct ways, most of us actually function as if our word consists only of _what I said_ or _what I assert_ _is true_. This guarantees that we cannot be a person of integrity. For us, Words 2, 3, 5, and 6 are invisible as our word:
```

>    - Word-2: What You Know to do or not to do

Reviewer: ```suggestion
   - *Word 2*: What you know to do or not to do
```

>    - Word-3: What Is Expected of you by those with whom you wish to have a workable relationship (unless you have explicitly declined those unexpressed requests)

Reviewer: ```suggestion
   - *Word 3*: What is expected of you by those with whom you wish to have a workable relationship (unless you have explicitly declined those unexpressed requests)
```

>    - Word-5: What You Stand For

Reviewer: ```suggestion
   - *Word 5*: What you stand for
```

>    - Word-6: Moral, Ethical and Legal Standards of each society, group, and governmental entity of which I am a member

Reviewer: ```suggestion
   - *Word 6*: Moral, ethical and legal standards of each society, group, and governmental entity of which I am a member
```

>    When we live (function in life) as though our word is limited to Word 1: What I Said and Word 4: What I say is so, we are virtually certain to be out of integrity with regard to our word as constituted in Words 2, 3, 5 and 6. In such cases, all the instances of our word (be it the word of an individual or organisation) that are not spoken or otherwise communicated explicitly are simply invisible as our word to such individuals or organisations. In our lives, all the instances of our Words 2, 3, 5 and 6 simply do not show up (occur) for us as our having given our word."

Reviewer: ```suggestion
   When we live (function in life) as though our word is limited to Word 1: What I Said and Word 4: What I say is so, we are virtually certain to be out of integrity with regard to our word as constituted in Words 2, 3, 5 and 6. In such cases, all the instances of our word (be it the word of an individual or organisation) that are not spoken or otherwise communicated explicitly are simply invisible as our word to such individuals or organisations. In our lives, all the instances of our Words 2, 3, 5 and 6 simply do not show up (occur) for us as our having given our word.
```

>    When we live (function in life) as though our word is limited to Word 1: What I Said and Word 4: What I say is so, we are virtually certain to be out of integrity with regard to our word as constituted in Words 2, 3, 5 and 6. In such cases, all the instances of our word (be it the word of an individual or organisation) that are not spoken or otherwise communicated explicitly are simply invisible as our word to such individuals or organisations. In our lives, all the instances of our Words 2, 3, 5 and 6 simply do not show up (occur) for us as our having given our word."

Reviewer: ```suggestion
   When we live as though our word is limited to word 1 and word 4, we are virtually certain to be out of integrity with regard to our word as constituted in words 2, 3, 5 and 6. In such cases, all the instances of our word (be it the word of an individual or organisation) that are not spoken or otherwise communicated explicitly are simply invisible as our word to such individuals or organisations. In our lives, all the instances of our words 2, 3, 5 and 6 simply do not occur for us as our having given our word."
```

Reviewer: This sentence is difficult to read. Can you word this differently and perhaps split up?

 _In such cases, all the instances of our word (be it the word of an individual or organisation) that are not spoken or otherwise communicated explicitly are simply invisible as our word to such individuals or organisations._

   **Example:**
   Sarah thinks integrity is just about what she says and what she claims is true. For example, she tells her team she'll attend meetings and believes being honest about project status is enough.

   But Sarah doesn't think about things she knows she should or shouldn't do (Word 2), what others expect from her (Word 3), her core values (Word 5), or ethical guidelines (Word 6). She misses deadlines even when she knows they're important (Word 2), ignores colleagues who expect her help with team tasks (Word 3), overlooks the importance of teamwork (Word 5), and doesn't follow company rules about transparency (Word 6).

   By focusing only on spoken words and claims, Sarah misses many parts of integrity. This makes people trust her less, even if she thinks she's being honest.

3. **"Integrity is a virtue"**

>    "For most people and organisations, integrity exists as a _virtue_ rather than as a _necessary condition for performance_. When held as a virtue rather than as a factor of production, integrity is easily sacrificed when it appears that a person or organisation must do so to succeed. For many people, virtue is valued only to the degree that it engenders the admiration of others, and as such it is easily sacrificed especially when

Reviewer: ```suggestion
   For most people and organisations, integrity exists as a _virtue_ rather than as a _necessary condition for performance_. When held as a virtue rather than as a factor of production, integrity is easily sacrificed when it appears that a person or organisation must do so to succeed. For many people, virtue is valued only to the degree that it engenders the admiration of others, and as such it is easily sacrificed especially when
```

   it would not be noticed or can be rationalised. Sacrificing integrity as a virtue seems

>    no different than sacrificing courteousness, or new sinks in the men's room."

Reviewer: ```suggestion
   no different than sacrificing courteousness, or new sinks in the men's room.
```

   **Example**:

   Tom runs a small business. He thinks integrity is just about looking good, like being polite. He doesn't see it as important for success.

   One day, Tom realizes he can make more money by breaking a small promise to a customer. He thinks it's okay because no one will notice, and making money seems more important.

   For Tom, integrity is just a nice quality, not something necessary for his business. So, he easily gives it up for a quick gain, just like he might skip being polite when no one's watching.

4. **Self-Deception about being out-of-integrity**

>    "People are mostly unaware that they have not kept their word. All they see is the 'reason', rationalisation or excuse for not keeping their word. In fact, people systematically deceive (lie to) themselves about who they have been and what they have done. As Chris Argyris concludes: "Put simply, people consistently act inconsistently, unaware of the contradiction between their espoused theory and their theory-in-use, between the way they think they are acting and the way they really act." And if you think this is not you, you are fooling yourself about fooling yourself. Because people cannot see their out-of-integrity behaviour, it is impossible for them to see the cause of the unworkability in their lives and organisations – the direct result of their own attempts to violate the Law of Integrity."

Reviewer: ```suggestion
   People are mostly unaware that they have not kept their word. All they see is the 'reason', rationalisation or excuse for not keeping their word. In fact, people systematically deceive (lie to) themselves about who they have been and what they have done. As Chris Argyris concludes: "Put simply, people consistently act inconsistently, unaware of the contradiction between their espoused theory and their theory-in-use, between the way they think they are acting and the way they really act." And if you think this is not you, you are fooling yourself about fooling yourself. Because people cannot see their out-of-integrity behaviour, it is impossible for them to see the cause of the unworkability in their lives and organisations – the direct result of their own attempts to violate the Law of Integrity.
```

   **Example**:

   Jane frequently arrives late to work, telling herself it's because of traffic delays. She doesn't realise she's not keeping her promise to be on time because she constantly makes excuses.

5. **Integrity is keeping one's word**

   "The belief that integrity is keeping one's word – period – leaves no way to maintain
   integrity when this is not possible, or when it is inappropriate, or when one simply
   chooses not to keep one's word. This leads to concealing not keeping one's word,
   which adds to the veil of invisibility about the impact of violations of the Law of
   Integrity."

   **Example:**
   Mark promises to help his friend move furniture on Saturday. When Saturday arrives, he's too tired and stays home without telling his friend. He believes he must always keep his promises but doesn't admit when he can't.

6. **Fear of acknowledging you are not going to keep your word**

>    "When maintaining your integrity (i.e., acknowledging that you are not going to keep

Reviewer: ```suggestion
   When maintaining your integrity (i.e., acknowledging that you are not going to keep
```

   your word and cleaning up the mess that results) appears to you as a threat to be
   avoided (like it was when you were a child) rather than simply a challenge to be dealt
   with, you will find it difficult to maintain your integrity. When not keeping their word,
   most people choose the apparent short-term gain of hiding that they will not keep their
   word. Thus out of fear we are blinded to (and therefore mistakenly forfeit) the power
   and respect that accrues from acknowledging that one will not keep one's word or that

>    one has not kept one's word."

Reviewer: ```suggestion
   one has not kept one's word.
```

   **Example:**

   Lisa promised to finish a project by Friday. When she realizes she can't, she doesn't tell her boss because she's afraid of disappointing him. This makes things more stressful for everyone.

7. **Integrity is not seen as a factor of production**

>    "This leads people to make up false causes and unfounded rationalisations as the

Reviewer: ```suggestion
   This leads people to make up false causes and unfounded rationalisations as the
```

   source(s) of failure, which in turn conceals the violations of the Law of Integrity as the

>    source of the reduction of the opportunity for performance that results in failure."

Reviewer: ```suggestion
   source of the reduction of the opportunity for performance that results in failure.
```

   **Example**:

   A company skips quality checks to produce faster, thinking integrity doesn't matter. When projects fail, they blame market trends instead of seeing that their lack of integrity caused the problems.

8. **_Not_ doing a cost/benefit analysis on _giving_ one's word**

>    "When giving their word, most people do not consider fully what it will take to _keep_ that word. That is, people do not do a cost/benefit analysis on giving their word. In effect, when giving their word, most people are merely sincere (well-meaning) or placating someone, and don't even think about what it will take to keep their word. Simply put, this failure to do a cost/benefit analysis on giving one's word is irresponsible. Irresponsible giving of one's word is a major source of the mess left in the lives of people and organisations. People generally do not see the giving of their word as: "I _am_ going to _make_ this happen," but if you are not doing this you will be out-of-integrity. Generally people give their word _intending_ to keep it. That is, they are merely sincere. If anything makes it difficult to deliver, then they provide _reasons_ instead of results."

Reviewer: ```suggestion
   When giving their word, most people do not consider fully what it will take to _keep_ that word. That is, people do not do a cost/benefit analysis on giving their word. In effect, when giving their word, most people are merely sincere (well-meaning) or placating someone, and don't even think about what it will take to keep their word. Simply put, this failure to do a cost/benefit analysis on giving one's word is irresponsible. Irresponsible giving of one's word is a major source of the mess left in the lives of people and organisations. People generally do not see the giving of their word as: "I _am_ going to _make_ this happen," but if you are not doing this you will be out-of-integrity. Generally people give their word _intending_ to keep it. That is, they are merely sincere. If anything makes it difficult to deliver, then they provide _reasons_ instead of results.
```

   **Example:**

   Tom says he'll help plan a friend's event but doesn't think about how busy he is. When his schedule gets too full, he can't help, causing disappointment and stress.

9. **_Doing_ a cost/benefit analysis on _honouring_ one's word**

>    "People almost universally apply cost/benefit analysis to _honouring_ their word. Treating integrity as a matter of cost/ benefit analysis guarantees you will not be a trustworthy person, or with a small exception, a person of integrity.

Reviewer: ```suggestion
   People almost universally apply cost/benefit analysis to _honouring_ their word. Treating integrity as a matter of cost/ benefit analysis guarantees you will not be a trustworthy person, or with a small exception, a person of integrity.
```

   If I apply cost/benefit analysis to honouring my word, I am either out of integrity to start with because I have not stated the cost/benefit contingency that is in fact part of my word (I lied), or to have integrity when I give my word, I must say something like the following:
   "I will honour my word when it comes time for me to honour my word if the costs of doing so are less than the benefits."

>    Such a statement, while leaving me with integrity will not engender trust. In fact it says that my word is meaningless."

Reviewer: ```suggestion
   Such a statement, while leaving me with integrity will not engender trust. In fact it says that my word is meaningless.
```

   **Example:**

   Sarah says she'll volunteer if she doesn't have anything else to do. When other things come up, she backs out, showing her promise wasn't sincere because it was based on convenience.

10. **Integrity is a Mountain with No Top**

>     "People systematically believe that they are in integrity, or if by chance they are at the

Reviewer: ```suggestion
    People systematically believe that they are in integrity, or if by chance they are at the
```

    moment aware of being out of integrity, they believe that they will soon get back into
    integrity.
    In fact integrity is a mountain with no top. However, the combination of 1) generally
    not seeing our own out-of-integrity behaviour, 2) believing that we are persons of
    integrity, and 3) even when we get a glimpse of our own out-of-integrity behaviour, assuaging ourselves with the notion that we will soon restore ourselves to being a
    person of integrity keeps us from seeing that in fact integrity is a mountain with no
    top. To be a person of integrity requires that we recognise this and "learn to enjoy

>     climbing"."

Reviewer: ```suggestion
    climbing".
```

    **Example**:

    Paul thinks he's always honest but overlooks the small lies he tells. He plans to fix them later, believing he can regain his integrity, not realizing that maintaining integrity is a continuous effort.

11. **Not having your word in existence when it comes time to keep your word**

>     "People say "Talk is cheap" because most people do not honour their word when it comes time to keep their word. A major source of people not honouring their word, is that when it comes time for them to do so, their word does not exist for them in a way that gives them a reliable opportunity to honour their word.

Reviewer: ```suggestion
    People say "Talk is cheap" because most people do not honour their word when it comes time to keep their word. A major source of people not honouring their word, is that when it comes time for them to do so, their word does not exist for them in a way that gives them a reliable opportunity to honour their word.
```

    Most people have never given any thought to keeping their word in existence so that when it comes time for them to keep their word there is a reliable opportunity for them to honour their word. This is a major source of out-of-integrity behaviour for individuals, groups and organisations.

>     In order to honour your word, you will need an extraordinarily powerful answer to the question, "Where Is My Word When It Comes Time For Me To Keep My Word?" \*\*\*\*If you don't have a way for your word to be powerfully present for you in the moment or moments that it is time for you to take action to honour your word, then you can forget about being a person of integrity, much less a leader."

Reviewer: ```suggestion
    In order to honour your word, you will need an extraordinarily powerful answer to the question, "Where Is My Word When It Comes Time For Me To Keep My Word?" \*\*\*\*If you don't have a way for your word to be powerfully present for you in the moment or moments that it is time for you to take action to honour your word, then you can forget about being a person of integrity, much less a leader.
```

>     In order to honour your word, you will need an extraordinarily powerful answer to the question, "Where Is My Word When It Comes Time For Me To Keep My Word?" \*\*\*\*If you don't have a way for your word to be powerfully present for you in the moment or moments that it is time for you to take action to honour your word, then you can forget about being a person of integrity, much less a leader."

Reviewer: ```suggestion
    In order to honour your word, you will need an extraordinarily powerful answer to the question, "Where Is My Word When It Comes Time For Me To Keep My Word?" If you don't have a way for your word to be powerfully present for you in the moment or moments that it is time for you to take action to honour your word, then you can forget about being a person of integrity, much less a leader."
```

I write down every task that comes to mind in the backlog. This means I have a lot of tasks! Every day after work, I look at my backlog and only move tasks to the to-do list if I've set a time for them in my calendar. Otherwise, they just add stress.

When you do this, you'll see how your commitments don't always fit into your schedule! When I find a time slot, I set a due date for the Trello task that matches the calendar event.

Important: I don't set deadlines like "Finish Task X." That adds stress, especially if there are unknowns. Instead, I plan actions: "Ask person X about documentation," "Work on feature X," or "Read article Y about feature X."

Tasks should be clear. The clearer they are, the less likely you are to procrastinate. This keeps your commitments organized.

Here's the key: I don't do anything that's not on my calendar. If I want to do something, it must be scheduled. I use phone notifications to remind me of my commitments.

This is the first step to keeping your word—if I realize I can't keep a promise (calendar events), I quickly tell those involved and reschedule.

> Tip: Not doing this can make you anxious about unfinished tasks. Rescheduling them helps you feel more productive and responsible.

Reviewer: ```suggestion
*Tip*: Not doing this can make you anxious about unfinished tasks. Rescheduling them helps you feel more productive and responsible.
```

You'll learn what a task is versus a deadline by using this method to avoid extra stress.

> Spend time every day organizing your Trello tasks and planning for the week.

Reviewer: ```suggestion
#### Spend time every day organizing your Trello tasks and planning for the week.
```

Writing about it helps me find contradictions in my promises. For example, I once promised to do something for my wedding over the weekend but didn't do it. Through journaling, I realized work-life balance is important to me. So, I moved the task to weekdays, which matched my goal of not using my laptop on weekends.

Sometimes, procrastination happens because you're not being true when you promise. For example, my manager asked me to do Task X, but I avoided it because I felt I shouldn't be doing it. I wasn't confident enough to say so. Fear of confrontation or unspoken feelings can lead to breaking your word.

Even if you don't try to find out what's really important to you, you might not be true to yourself because you're pretending to be something you're not—not out of dishonesty but because you don't know what you really want. This leads to promising things you don't want to do.

Also, if you're not sure how to do a promised task, you might need to plan some steps first. It can help to schedule a task just to figure out what's expected of you. Sometimes, you might even need a task before that one. Keep organizing until your first step is clear and doable.

> Tip: if there's a task that takes less than 2 minutes (like messaging someone), I do it during planning.

Reviewer: ```suggestion
*Tip*: if there's a task that takes less than 2 minutes (like messaging someone), I do it during planning.
```
